packages/reports/app/styles/navi-reports/components/navi-list-selector.less

```diff
@@ -71,14 +71,8 @@
     padding: 12px 24px;
     text-align: left;
     transition: 200ms;
-    width: inherit;
-
-    &:active,
-    &:focus {
-      border-bottom-color: @denali-brand-700;
-      border-bottom-style: solid;
-      border-bottom-width: 2px;
-    }
+    width: 230px;
```

Comment:
Can most of the styles here now be removed? Also, can we clean up `&__search-icon`?